requirements.txt

```diff
@@ -1,8 +1,8 @@
+AltTester-Driver==1.8.0
 selenium
 pytest
 pytest-selenium
```

Comment:
I don't think we use `selenium`, `pytest-selenium`, `pytest-html`, or `assertpy` so we should remove them from `requirements.txt`.